### DISCUSSION TITLE
Could you tell me your favorite music?

### DISCUSSION THREAD

**Discussion by marimo-n2525** [NAIST-SE/AttractingContributors#927] [2023 Attraction Contributions]
my favorite music genre is "kawaii future bass" and "city pop".
if you have music that you can recommend, please tell me.

repo:
https://github.com/NAIST-SE/AttractingContributors

**Comment by kkkyk373** [comment]
I think your link is wrong.
Is the correct one this https://github.com/marimo-n2525/AttractingContributors?
